key,default_value
userAgent,"Boto3/1.10.19 Python/3.6.8 Linux/4.15.0-66-generic Botocore/1.13.19"
eventID,"c23d9ff2-771d-49ea-a6c9-8f3db078d74e"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: ds:DescribeTrusts"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"4.237.1.155"
eventName,"DescribeTrusts"
eventSource,"ds.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"e362ba94-4b96-44fa-bc13-38655a8ea190"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2019-11-20T07:43:12Z"
